CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -91,5 +90,38 @@ public void CanRemindAUser()
 
             ThenMessageHasBeenPostedToSlack("U172L982");
         }
+
+        [Test]
+        public void CanGetUsersFromHarvest()
+        {
+            var harvestGetUsersResponse = new HarvestGetUsersResponse
+            {
+                Success = true
+            };
+
+            var getUsers = new GetUsers(new HarvestGateway(
+                "http://localhost:8009/",
+                "xxxx-xxxxxxxxx-xxxx"
+            ));
+            var response = getUsers.Execute();
+            
+            _fluentSimulator.Get("/api/v2/users").Responds(harvestGetUsersResponse);
+            
+            var receivedRequest = _fluentSimulator.ReceivedRequests.First();
+
+            receivedRequest.Url.Should().Be(
+                "http://localhost:8009/api/v2/users"
+            );
+            receivedRequest.Headers["Authorization"].Should().Be(
+                "Bearer xxxx-xxxxxxxxx-xxxx"
+            );
+
+            DeserializeObject<HarvestGetUsersResponse>(response).Success.Should().Be(true);
+        }
+    }
+
+    public class HarvestGetUsersResponse
```

Comment:
I think we should move this up, just below `SlackPostMessageResponse` so they're together.

Reply:
done